setup.py

```diff
@@ -44,19 +55,23 @@ def package_files(directory):
 
 
 setup(
-    name="pca-report-library",
+    name="pca_report_library",
```

Comment:
The name of the package can have hyphens and this aligns with the repository name. It will still import as `pca_report_library`.

```suggestion
    name="pca-report-library",
```

Reply:
Resolved in f478d7c

Reply:
Resolved in a1252af